docusaurus/docs/develop/envoy/introduction.md

```diff
@@ -404,7 +482,9 @@ The custom implementation must use the methods defined in the `GatewayEndpoints`
 - `FetchAuthDataSync`
 - `StreamAuthDataUpdates`
 
-> 💡 **TIP:** Forking the PADS repo is the easiest way to get started, though any gRPC server implementation that adheres to the `gateway_endpoint.proto` service definition should suffice.
+:::tip
+Forking the PADS repo is the easiest way to get started, though any gRPC server implementation that adheres to the `gateway_endpoint.proto` service definition should suffice.
```

Comment:
Should they fork it in both the YAML and posters use-cases or just the postgres one?

#PUC